README.md

```diff
@@ -91,6 +92,31 @@ is provided by code that been generated from a model of the service.
 * [Contributions Guidelines](./documents/CONTRIBUTING.md)
 * [DEVELOPING](./documents/DEVELOPING.md)
 
+## FIPS Compliance
+
+This SDK supports [FIPS compliance](https://aws.amazon.com/compliance/fips/) on *some platforms*. If you're on a FIPS-compliant
+platform, FIPS will be enabled by default. See the [Platform-Specific JARs section](https://github.com/awslabs/aws-crt-java?tab=readme-ov-file#platform-specific-jars)
```

Comment:
update the link to https://github.com/awslabs/aws-crt-java?tab=readme-ov-file#fips-compliance and corresponding name